dwitter/serializers_v2.py

```diff
@@ -100,6 +100,7 @@ class DweetSerializer(serializers.ModelSerializer):
     id = serializers.ReadOnlyField(source='pk')
 
     remix_of = RemixOfSerializer(source='reply_to')
+    remixes = serializers.PrimaryKeyRelatedField(source='dweet_set', read_only=True, many=True)
```

Comment:
We use this serializer to list dweets right? Perhaps we want to add prefetch_related for the dweet_set if we haven't already, for performance.

Reply:
The current setup doesn't ask for any of the dweets in this list unless a user explicitly clicks the remix list; which I expect is infrequent. Do you still think it's worth doing a prefetch for every request?